packages/synapse-sdk/src/sp-registry/service.ts

```diff
@@ -423,46 +396,27 @@ export class SPRegistryService {
    * @returns PDP service info or null if not found
    */
   async getPDPService(providerId: number): Promise<PDPServiceInfo | null> {
-    try {
+    //try {
```

Comment:
this is the main reason I haven't ran lint yet. Hoping to keep this removed until the tests pass, as otherwise it might mask some errors with the new code.